.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
+!dist/node_modules/
```

Comment:
**question**: is this because esm doesn't bundle?

Reply:
Yes, it's because I couldn't get the bundling to work. I tried esbuild, ncc, and rollup.